docs/source/tutorial/CP_APR.ipynb

```diff
@@ -107,7 +100,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.15"
+   "version": "3.11.2"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "b0fa6594d8f4cbf19f97940f81e996739fb7646882a419484c72d19e05852a7e"
+   }
```

Comment:
@DeepBlockDeepak this vscode meta data appears to break myst_nb. If we look at the runtime from when I first added the preview it takes about 2s. If I remove this vscode snippet locally the build works for me again.

Reply:
Related to my note earlier about needing to resolve clearing extra information from our notebooks from the repo. If `nbstripout` pulls out the vscode meta data then we might already know the approach. I am kind of surprised this causes things to fall over so hard.

Reply:
Hmm! It is actually more interesting. I ran nbstripout and it only changed line ending. I think there is a bad whitespace character that the myst_nb parser is barfing on but regular jupyter figures out.

Reply:
I think when I locally removed that snippet when I save it it reflowed the file to linux style LF, where previously it was CRLF

Reply:
@DeepBlockDeepak I think you are developing in Windows, is that correct? Can you check your git line endings and verify that you are not forcing all your commits to use CRLF endings? If this is. It the issue, we should try to determine what is.

Reply:
> Hmm! It is actually more interesting. I ran nbstripout and it only changed line ending. I think there is a bad whitespace character that the myst_nb parser is barfing on but regular jupyter figures out.

nbstripout has option to remove any set of metadata you specify with  —extra-keys parameter. https://github.com/kynan/nbstripout

Reply:
I'm developing in MacOS @dmdunla . I did run `git config --get core.autocrlf` and no value was set or returned to me. I'm guessing that's a good thing.